src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/inject/Injector.kt

```diff
@@ -21,7 +21,13 @@ object Injector {
     val directoryProvider = persistenceComponent.injectDirectoryProvider()
     val resourceContainerDirectory = directoryProvider.resourceContainerDirectory
     val languageRepo = LanguageRepository(database, LanguageMapper())
-    val collectionRepo = CollectionRepository(database, CollectionMapper(), ResourceMetadataMapper(), LanguageMapper())
+    val collectionRepo = CollectionRepository(
+            database,
+            CollectionMapper(),
```

Comment:
Why don't we move directoryProvider up in the arguments list, and allow the default arguments to construct the Mappers